pkg/orchestrator/github_actions.go

```diff
@@ -109,7 +109,7 @@ func (g *githubActionsConfigProvider) FullLogs() ([]byte, error) {
 	for i := range jobs {
 		i := i // https://golang.org/doc/faq#closures_and_goroutines
 		wg.Go(func() error {
-			_, resp, err := g.client.Actions.GetWorkflowJobLogs(g.ctx, g.owner, g.repo, jobs[i].ID, true)
+			_, resp, err := g.client.Actions.GetWorkflowJobLogs(g.ctx, g.owner, g.repo, jobs[i].ID, 1)
```

Comment:
The last parameter denotes the maximum number of redirects the github client is allowed to follow. Up to now this was hardcoded to `1`. Since we never had issues with it, I decided to keep it at this level.